standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -619,15 +666,15 @@ private void setCleanerStart(Connection dbConn, CompactionInfo info, Long timest
    */
   @Override
   @RetrySemantics.CannotRetry
-  public void markCleaned(CompactionInfo info, boolean isAbortOnly) throws MetaException {
+  public void markCleaned(CompactionInfo info) throws MetaException {
     LOG.debug("Running markCleaned with CompactionInfo: {}", info);
     try {
       Connection dbConn = null;
       PreparedStatement pStmt = null;
       ResultSet rs = null;
       try {
         dbConn = getDbConn(Connection.TRANSACTION_READ_COMMITTED, connPoolCompaction);
-        if (!isAbortOnly) {
+        if (!info.isAbortedTxnCleanup()) {
```

Comment:
we no longer generate completed_compactions entries for the aborts, right?

Reply:
Yes, we no longer generate completed_compactions entries.

Reply:
previously in case of unexpected situation, we were able to figure out the affected tables from the DB, now we'll have to grep the logs. should be fine, right?